main/main_timer_sync.cpp

```diff
@@ -307,18 +307,21 @@ double MainTimerSync::get_physics_jitter_fix() {
 int MainTimerSync::get_average_physics_steps(double &p_min, double &p_max) {
 	p_min = typical_physics_steps[0];
 	p_max = p_min + 1;
+	int i;
+	int iplus1;
```

Comment:
What are you basing the idea that moving a declaration out of scope improves performance?